CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 ### New features and optimizations
 - Add MoE Support for our reward models.
 - SFT/SteerLM: LoRA can now be enabled on all model layers
+- DPO: Enable LoRA on all model layers
```

Comment:
I would recommend we add a comment here about how there is no reference model when using LoRA (as far as I can tell from the code) and explain the reasoning behind how that is designed to work

Reply:
Added https://github.com/NVIDIA/NeMo-Aligner/pull/142/commits/4237f5eb311fc097ce5b032e3cba5df7201c70cd